packages/spor-theme-react-native/src/foundations/colors.tsx

```diff
@@ -7,6 +7,13 @@ const colorAliases = Object.entries(tokens.color.alias).reduce(
   {} as { [key in keyof typeof tokens.color.alias]: string }
 );
 
+const colorLinjetags = Object.entries(tokens.color.linjetag).reduce(
+  (acc, [key, token]) => {
+    return { ...acc, [`linjetag.${key}`]: token.value };
+  },
+  {} as { [key in `linjetag.${keyof typeof tokens.color.linjetag}`]: string }
+);
+
```

Comment:
Denne ekstraherer linjetag-fargene fra design tokens og tilgjengeliggjør dem for restyle som `linjetag.fargenavn`.